src/Symfony/Component/DomCrawler/Field/ChoiceFormField.php

```diff
@@ -143,14 +143,14 @@ public function setValue(string|array|bool|null $value): void
      */
     public function addChoice(\DOMElement $node): void
     {
-        if (!$this->multiple && 'radio' !== $this->type) {
+        if (!$this->multiple && ('radio' !== $this->type && 'select' !== $this->type)) {
             throw new \LogicException(sprintf('Unable to add a choice for "%s" as it is not multiple or is not a radio button.', $this->name));
```

Comment:
Message (and method docblock) should be updated. 

@stof  Would that be a BC break, as select were refused before ?

Reply:
@smnandre given that's an `@internal` method BC break does not apply here